vispy/geometry/generation.py

```diff
@@ -89,6 +89,250 @@ def create_cube():
     return vertices, filled, outline
 
 
+def create_plane(width=1,
+                 height=1,
+                 width_segments=1,
+                 height_segments=1,
+                 direction='+z'):
```

Comment:
we try to fit as many of these on the same line as possible while still being PEP8